### PR TITLE
how to deploy the server on heroku

### PR DESCRIPTION
**Pull request by anicacortes**

### CODE REVIEW

File: description.md [hunk 1]
```diff
@@ -20,16 +20,16 @@ JUNIT TEST FRAMEWORK:
...
--an existing Java app That Gradle use as a build tool.
--a free Heroku account
--Heroku the CLI
-If you are using gradlew, then you must also add your gradle/wrapper/gradle-wrapper.jar and gradle/wrapper/gradle-wrapper.properties to your Git repository.
-
+- an existing Java app that uses Gradle as a build tool.
+- a free Heroku account
+- Heroku the CLI
+If you are using gradlew, then you must also add your `<gradle/wrapper/gradle-wrapper.jar>` and `<gradle/wrapper/gradle-wrapper.properties>` to your Git repository.
+[BIBLIOGRAPHY] ()https://devcenter.heroku.com/articles/deploying-gradle-apps-on-heroku)
```
Comment (RecuencoJones) [inline description.md]: the link syntax is broken:

[BIBLIOGRAPHY](https://devcenter.heroku.com/articles/deploying-gradle-apps-on-heroku)

**Comment by RecuencoJones** [comment]
No need to close the pull request, you can just add the commits here!

**Comment by RecuencoJones** [comment]
Looks good to me!

LGTM

**Comment by fjlopez** [comment]
Hum... Requires a bit of work for merging but its Ok.  I can assume that.

**Pull request merged**